defaults/main.yml

```diff
@@ -639,6 +655,11 @@ postgresql_max_locks_per_transaction: 64         # min 10
 # lock table slots.
 postgresql_max_pred_locks_per_transaction: 64    # min 10
 
+postgresql_max_pred_locks_per_relation: -2       # (>= 10) negative values mean
+               # (max_pred_locks_per_transaction
```

Comment:
Do we need these comments in this file?  If someone wants to know what it does, then they can check the doco, or the `postresql.conf` itself.

Reply:
Right. Same for other comments taken from the original config file.  May be for less exerienced people like me helpful to read it here too :-).